integration/loki_micro_services_delete_test.go

```diff
@@ -160,6 +109,37 @@ func TestMicroServicesDeleteRequest(t *testing.T) {
 		assert.ElementsMatch(t, []string{"lineA", "lineB", "lineC", "lineD"}, lines)
 	})
 
+	t.Run("flush-logs-and-restart-ingester-querier", func(t *testing.T) {
+		// restart ingester which should flush the chunks
+		require.NoError(t, tIngester.Restart())
+		// ensure that ingester has 0 chunks in memory
+		cliIngester = client.New(tenantID, "", tIngester.HTTPURL())
+		cliIngester.Now = now
+		metrics, err := cliIngester.Metrics()
+		require.NoError(t, err)
+		checkMetricValue(t, "loki_ingester_memory_chunks", metrics, 1)
```

Comment:
This assertion contradicts with the comment on line 115. Shouldn't there be no chunks in memory?

Reply:
sorry, my bad. I have updated the metric.
The value is 1 because the metric is lazily registered however the registry still has the previous metric value since it is global.